src/guards/role.guard.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  CanActivate,
+  Injectable,
+  ExecutionContext,
+  UnauthorizedException,
+  Logger,
+  Inject,
+} from '@nestjs/common'
+
+import { KeycloakService } from '../service'
+import { Reflector } from '@nestjs/core'
+import { META_PUBLIC } from '../decorators/public.decorator'
+import { META_ROLES, RoleDecoratorOption } from '../decorators/roles.decorator'
+import { RoleMatchingMode } from '../constants'
+import KeycloakConnect from 'keycloak-connect'
+
+@Injectable()
+export class Roleuard implements CanActivate {
```

Comment:
Can you fix the typo in here? It should be `RoleGuard` not `Roleuard`

Reply:
will do